ACKategories-iOS/SelfSizingTableHeaderFooterView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  SelfSizingTableHeaderFooterView.swift
+//  ACKategories-iOS
+//
+//  Created by Karel Leinhäupl on 06.09.2020.
+//
+
+import UIKit
+
+/// This view will autolayout its height, even when used as a tableHeaderView or tableFooterView.
+open class SelfSizingTableHeaderFooterView: UITableViewHeaderFooterView {
+
+    fileprivate enum Status {
```

Comment:
do all the properties in this class need to be `fileprivate`? Seems to me that `private` might make more sense here

Reply:
Yes, you are right. It doesn't make sense to use a fileprivate modifier in this class.